.github/workflows/release-new-version.yml

```
@@ -74,7 +74,11 @@ jobs:
             connector=$(basename ${connector_dir})
             newman dir-import ${POSTMAN_DIR}/${connector} -o ${POSTMAN_JSON_DIR}/${connector}.postman_collection.json
           done
-          (git diff --quiet && git diff --staged --quiet) || (git commit -am 'test(postman): update postman collection files' && echo "Committed changes") || (echo "Unable to commit the following changes:" && git diff)
+
+          if git add postman && ! git diff --staged --quiet postman; then
+            git commit -am 'test(postman): update postman collection files'
```

```suggestion
            git commit --message 'test(postman): update postman collection files'
```

Sufficient to commit changes, since we've already staged changes in the `postman` directory.